spec/factories/comments.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+	factory :comment do
```

Comment:
please re-read ruby style guide.
we use 2 spaces (not tabs) for identation. please dont user python-style